services/api/routes/authenticated.js

```diff
@@ -169,10 +169,7 @@ var routes = [
           project: numericSchema
         },
         payload: {
-          title: Joi.string().optional(),
-          thumbnail: Joi.object().keys({
-            320: Joi.string().optional()
-          }).default({})
+          title: Joi.string().required()
```

Comment:
What happens if I send a payload with a thumbnail? Will it give me a validation error? All of the current versions of the Android app will send the thumbnail along